_inc/connect-button.js

```diff
@@ -172,7 +173,11 @@ jQuery( document ).ready( function ( $ ) {
 	};
 
 	// When we visit /wp-admin/admin.php?page=jetpack#/setup, immediately start the connection flow.
-	var hash = location.hash.replace( /#\//, '' );
+	var hash = location.hash.replace( /(#\/setup).*/, 'setup' );
```

Comment:
I did this to clean out the variable for the comparison below. I don't think we use it for anything other than `setup`, but correct me if I'm wrong.